modules/op-uninstalling-the-pipelines-operator.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/uninstalling-pipelines.adoc
+
+[id='op-uninstalling-the-pipelines-operator_{context}']
+= Uninstalling the {pipelines-title} Operator
+
+[discrete]
+== Procedure
+. From the *Operators->OperatorHub* page, use the *Filter by keyword* box to search for {pipelines-title} Operator.
+
+. Click on the *OpenShift Pipelines Operator* tile. The Operator tile indicates it is installed.
+
+. In the *OpenShift Pipelines Operator* descriptor page, click **Uninstall**.
+
+. In the *Remove Operator Subscription* dialog box, ensure that the *Also completely remove the Operator from the selected namespace* is selected, and click *Remove*.
```

Comment:
I don't see anything after I clicked `Uninstall` in previous step

Reply:
removed this step